app/presenters/hyrax/manifest_components.rb

```diff
@@ -0,0 +1,77 @@
+module Hyrax
+  module ManifestComponents
+    def manifest_url
+      manifest_helper.polymorphic_url([:manifest, self])
+    end
+
+    # IIIF rendering linking property for inclusion in the manifest
+    #
+    # @return [Array] array of rendering hashes
+    def sequence_rendering
+      renderings = []
+      if solr_document.rendering_ids.present?
+        solr_document.rendering_ids.each do |file_set_id|
+          renderings << build_rendering(file_set_id)
+        end
+      end
+      renderings.flatten
+    end
+
+    # IIIF metadata for inclusion in the manifest
+    #
+    # @return [Array] array of metadata hashes
+    def manifest_metadata
+      metadata = []
+      metadata_fields.each do |field|
+        metadata << {
+          'label' => field.to_s.humanize.capitalize,
```

Comment:
Do we need string keys here or can we use Ruby 1.9-style hash syntax?

Reply:
I *think* iiif_manifest wants them this way ... that could be changed, but the @id syntax would be more problematic as you can't give that as a symbol AFAIK ... I'm minded to leave them like this, unless you feel strongly about this and I can do more digging

Reply:
K, sounds fine to me.

Reply:
Yep, sounds good to me. I'll try to get this wrapped up tomorrow.

Reply:
Can we use i18n for the field labels?